### ISSUE TITLE
Sequence of actions

### ISSUE THREAD

**Issue by jfernan123** [nguyen-thanh05/Deep-Reinforcement-Learning-agent-for-Construction-Project-Execution#7]
Add capacity to record sequence of actions taken. Pick format for later rendering

**Comment by mac-op** [comment]
Yeah we need a standardized format to serialize/deserialize actions and building data. Could use a binary format like raw bytes or protobuf, or if both the renderer and model use Python then .npy format. To be decided later
